docs/core/compatibility/sdk/6.0/write-reference-assemblies-to-obj.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Breaking change: Write reference assemblies to IntermediateOutputPath"
+description: Learn about the breaking change in .NET 6 where reference assemblies are written to the IntermediateOutputPath by default.
+ms.date: 07/13/2021
+---
+# Write reference assemblies to intermediate output
+
+The .NET SDK now writes [reference assemblies](https://docs.microsoft.com/dotnet/standard/assembly/reference-assemblies) to the `IntermediateOutputPath` instead of the `OutDir` by default. This removes these build-time-only artifacts from your runtime-required outputs.
+
+## Version **introduced**
+
+.NET SDK 6.0.200
+
+## Old behavior
+
+Since reference assemblies were added, the .NET SDK has written reference assemblies to the `ref` directory in the `OutDir` of the compilation. 
```

Comment:
```suggestion
Since reference assemblies were added, the .NET SDK has written reference assemblies to the `ref` directory in the `OutDir` directory of the compilation.
```